README.md

```diff
@@ -155,46 +181,72 @@ To view the app, open the URL in your browser.
+$ export POD_NAME=$(kubectl get pods \
+       -l "app=conjur-oss" \
+       -o jsonpath="{.items[0].metadata.name}")
+$ kubectl exec $POD_NAME --container=conjur-oss conjurctl account create default
 Token-Signing Public Key: -----BEGIN PUBLIC KEY-----
-MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAv9+iXUFZISHlZfGrkio5
-RG4hPijORLLMwNlHx7Bv0NMG2gFpwS25xDmBK2UgARlow+b9OS9spmLAdH15lXP5
-40HBkKX/aYnLCH55TC4T/GslyUiaLZocVn1nkExdnBYO+tbNjUUJnHp2tljfQKCD
-Lp+uqDlwbPDzUEqxPRj70ro0F8aeCVvjb0AQ7dHMnZ/FYYpJShxOwtn/FeaC+mhU
-85QlcrtN3kn+pAA9wxuoKpIN8DoCK506AftL5Dra9xdconneSZ4XhaweAzut0BLp
-Oi5nZFPVNXAUBJAg/RmmgO2C3J8zBS66wo3L7XAhJs/TXzhKYxHQKNa6GoWgc7uW
-RwIDAQAB
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA06mdosQTY76NLQTrgr7v
+jkNLZC/a9jiKgeRTSJkMf+nJBLOVGmGgSZeU+eqDs/1Ldz/XJLouRk6XbSR8kAAQ
+FtZbXFQKyyrRAJg3jN9DbB930FfyuBHpI/dPZVmKbBqiL4P8pwW9oj5ACzBgB1ZF
+yz5iDWbmNyvIaqoYvSKpB7PItISOSX7C88LtxDsPK+eMxQnlu2kEg++P7OG2SFSW
+EpVAd8v13QOUTG8u7dJ8LRJDBt7cBMagGAxp+cTRxvIGp63joBbn8Ca9rhZBMaeT
+i/cFSx2B05QepUEFTVIJtSyF6cLUnRiXnZXVk61aRNbWOTEK8dGvkIBFswXPAN8z
+/QIDAQAB
 -----END PUBLIC KEY-----
-API key for admin: r41crc30ys1hv3njzrvz30xq6k210ec4y61y5qmmj1xyb300btrxmer
+Created new account 'default'
+API key for admin: 1ma6hxgt6fagm52qgtn344xd1v1b7qrgp571fsm1250z6r3aewb9t
+$
 ```
 
 > Note that the `conjurctl account create` command gives you the public key and admin API key for the account you created. Back them up in a safe location.
 
-Run `kubectl get svc "$APP_INSTANCE_NAME-conjur"` until the `EXTERNAL-IP` column resolves.
 
 ### Connect remote with the Conjur CLI
 
-Now pull the latest [cyberark/conjur-cli:5 image](https://hub.docker.com/r/cyberark/conjur-cli/) and run it to connect to Conjur.
+Fetch the external IP for the Conjur service:
+
+```shell
+INGRESS_SVC=$(kubectl get svc --no-headers -o custom-columns=":metadata.name" | grep conjur-oss-ingress)
+EXT_IP=$(kubectl get svc $INGRESS_SVC -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+Pull and run the latest [cyberark/conjur-cli:5 image](https://hub.docker.com/r/cyberark/conjur-cli/) to connect to Conjur:
+
+```shell
+docker pull cyberark/conjur-cli:5
+docker run --rm -it --env EXT_IP=$EXT_IP --entrypoint bash cyberark/conjur-cli:5
+```
+
+> Note that when connecting to the Conjur server, you must use hostname that matches one of the subject names that are contained in Conjur server's CA certificate, or you will get errors trying to log in. By default, the Conjur OSS helm charts will generate a CA certificate for the Conjur server that uses a DNS common name of `conjur.myorg.com`, so use that as a hostname here.
+
+```shell
+COMMON_NAME=conjur.myorg.com
+```
 
-> Note that you must use the DNS name you used in your deployment to connect to Conjur instead of the IP or you will get errors trying to log in.
+Set up a DNS A record for the target hostname `conjur.myorg.com`, or
+alternatively, create a mapping entry in /etc/hosts:
 
+```shell
+echo "$EXT_IP $COMMON_NAME" >> /etc/hosts
```

Comment:
Ditto about `sudo` and noting that this could create duplicates and earlier entries will override newer ones. I think we may need a better CLI addition of a host.

Reply:
Changed this to first look for an existing entry for the common name. If an entry exists, it is overwritten with a new external IP with `sed`. Otherwise, a new entry is added to /etc/hosts.